dot-gen/src/main/java/com/wipro/ats/bdre/ldg/LineageDotGen.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.wipro.ats.bdre.ldg;
+
+import com.wipro.ats.bdre.BaseStructure;
+import com.wipro.ats.bdre.GetLineageQueryByProcessId;
+import com.wipro.ats.bdre.lineage.LineageConstants;
+import com.wipro.ats.bdre.lineage.LineageMain;
+import com.wipro.ats.bdre.md.dao.jpa.LineageQuery;
+import org.apache.commons.cli.CommandLine;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+/**
+ * Created by AshutoshRai on 1/18/16.
+ */
+public class LineageDotGen extends BaseStructure {
+
+    protected static final Logger logger = LoggerFactory.getLogger(LineageDotGen.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1312)